README.md

```diff
@@ -507,6 +507,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 *  [Sequel Ace](https://github.com/Sequel-Ace/Sequel-Ace) - The maintained "sequel" to the longtime macOS tool Sequel Pro. [![Open-Source Software][oss icon]](https://github.com/Sequel-Ace/Sequel-Ace)
 * [SQLight](https://aurvan.com/sqlight/) - SQLite database manager tool. ![Freeware][Freeware Icon]
 * [SQLPro Studio](http://www.sqlprostudio.com/) - Simple, powerful database manager for macOS.
+* [Studio 3T](https://studio3t.com/) - the Ultimate GUI for MongoDB. ![Freeware][Freeware Icon]
```

Comment:
It's not free.

@aalfath

Reply:
Hi @jaywcjlove, it actually has a free edition as described here: https://studio3t.com/free/

Shall I change the URL to that specific link instead?

Thanks for checking out this PR!

Reply:
@aalfath Is this its purchase interface? https://studio3t.com/buy/

Reply:
@jaywcjlove that's the purchase page for the non-free edition. the free one does not need to be purchased.